### PR TITLE
A little better support for Windows Clang

### PR DESCRIPTION
**Pull request by jwrober**
... And other things.

Relies on #2098 to be merged first.

### CODE REVIEW

File: cmake/FreecivInstall.cmake [hunk 1]
```diff
@@ -127,25 +127,23 @@ elseif(WIN32)
   # The Visual Studio generator places all files and associated DLL libraries
   #  into a build directory. So we just grab those for install.
   install(
-    DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/
+    DIRECTORY ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/debug/bin/
```
Comment (lmoureaux) [inline cmake/FreecivInstall.cmake:130]: Does this still work if you ask for a release build instead of a debug build?

Comment (jwrober) [reply cmake/FreecivInstall.cmake:130]: Yes it does.

Comment (lmoureaux) [reply cmake/FreecivInstall.cmake:130]: Strange, I wouldn't expect a `release` build to put files in a `debug` folder (did you start the release build in a fresh build folder?).

**Review by lmoureaux — APPROVED** [review]

File: cmake/FreecivInstall.cmake (hunk 1 above)
Comment (jwrober) [reply cmake/FreecivInstall.cmake:130]: LoL. Its Visual Studio + vcpkg + Windows craziness

**Comment by jwrober** [comment]
Looks like I need to rebase.

**Comment by lmoureaux** [comment]
> Looks like I need to rebase.

Or we do a merge commit

**Pull request merged**